ablySpec/RestClientStats.swift

```diff
@@ -312,8 +312,12 @@ class RestClientStats: QuickSpec {
                             query.start = NSDate.distantFuture()
                             query.end = NSDate.distantPast()
 
-                            let error = client.stats(query, callback:{ status, result in })
-                            expect(error).toNot(beNil())
+                            do {
+                                try client.stats(query, callback:{ status, result in })
```

Comment:
You can _expect_ it like: `expect{ try somethingThatThrows() }.to(throwError())` or even with

```
expect{ try somethingThatThrows() }.to(throwError { (error: ErrorType) in
    expect(error._domain).to(equal(NSCocoaErrorDomain))
})
```